tests/drum/test_unstructured_mode.py

```diff
@@ -0,0 +1,114 @@
+import glob
+import json
+import os
+import re
+from tempfile import NamedTemporaryFile
+
+from uuid import uuid4
+
+import numpy as np
+import pandas as pd
+import pytest
+import requests
+
+from .constants import TESTS_ROOT_PATH
+from .utils import (
+    _exec_shell_cmd,
+    _cmd_add_class_labels,
+    _create_custom_model_dir,
+)
+from .drum_server_utils import DrumServerRun
+
+from datarobot_drum.drum.common import (
+    ArgumentsOptions,
+    CUSTOM_FILE_NAME,
+    CustomHooks,
+    PythonArtifacts,
+    RunMode,
+)
+from datarobot_drum.drum.runtime import DrumRuntime
+from datarobot_drum.drum.utils import CMRunnerUtils
+
+from .constants import (
```

Comment:
We'll probably want black on this repo? I think most of the imported constants are not used

Reply:
*I meant pylint

Reply:
done